packages/cozy-client/examples/helpers/file.js

```diff
@@ -0,0 +1,116 @@
+const { default: CozyClient, QueryDefinition } = require('cozy-client')
+const fs = require('fs')
+const { FileDocument } = require('cozy-client/dist/types')
+
+const schema = {
+  files: {
+    doctype: 'io.cozy.files'
+  }
+}
+
+const generateFakeFolder = () => ({
+  name: 'folder-name' + Math.random(),
+  icon: 'icon',
+  type: 'directory',
+  dirId: 'io.cozy.files.root-dir'
+})
+
+const generateFakeFile = (folder, i, file) => ({
+  name: `${folder.name}-${i}.txt`,
+  contentType: 'text/plain',
+  icon: 'icon',
+  path: `/${folder.name}/name${i}.txt`,
+  type: 'file',
+  dirId: folder._id,
+  data: file
+})
+
+const createFolder = async client => {
+  const { data: folder } = await client.stackClient
+    .collection('io.cozy.files')
+    .create(generateFakeFolder())
+  return folder
+}
+
+const createFileInsideFolder = async (client, count, folder) => {
+  const readme = fs.readFileSync('../README.md')
```

Comment:
This makes an assumption on the current path,  I don't think it will work if we do `node packages/cozy-client/examples/delete-files.js`
`readFileSync` returns a Buffer, so maybe `const fakeFile = Buffer.from('this is a test')` might work?